_data/navigation.yml

```
@@ -1,15 +1,17 @@
-- title: RSE Service
+- title: Work With RSEs
```

```suggestion
- title: The RSE Collaborative Model
```
It needs to give the impression that we do/deliver something (without using the word service)

---

I want the link to be a call to action - for people to look at the home page and go "What's next?" and then click it. It could be "Collaborate"?

---

Yes just collaborate would work